From 7ffa68d88c7a60403eb92c9ba606c8656c4fa0b3 Mon Sep 17 00:00:00 2001
From: Virendra Pathak <vpathak@broadcom.com>
Date: Thu, 11 Jul 2013 14:53:51 +0530
Subject: [PATCH 0385/1532] poe: enabled distribution mode by default

- enabled poe distribution by default
- changed the default complex mode to xaui for complex 0 and complex 1
- updated the xaui parameters for complex 0 and complex 1
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/xlp9xx-helinux.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/mips/boot/dts/xlp9xx-helinux.dts b/arch/mips/boot/dts/xlp9xx-helinux.dts
index 8992e6d..e9459f5 100644
--- a/arch/mips/boot/dts/xlp9xx-helinux.dts
+++ b/arch/mips/boot/dts/xlp9xx-helinux.dts
@@ -43,11 +43,11 @@
 
 			cpu {
 				onlinemask = <0x1 0x0 0x0>;
-                                nae-rx-vc = <1>;
+                                nae-rx-vc = <0>;
                                 nae-fb-vc = <1>;
-				sae-rx-vc = <1>; /* sharing with nae */
+				sae-rx-vc = <0>; /* sharing with nae */
 				sae-rx-sync-vc = <3>; /* should be exclusive */
-                                napi-vc-mask = <0x2>;
+                                napi-vc-mask = <0xb>;
 
 			};
 			uart {
@@ -203,7 +203,7 @@
 	};
 
 	chosen {
-		netlogic,cpumask = <0x01>;
+		netlogic,cpumask = <0xffff>;
 		bootargs = " console=ttyS0,115200  ";
 	};
 
-- 
1.9.1

